### PR TITLE
Test super-linter workflow

### PR DESCRIPTION
**Pull request by mmacata**
Make use of [super-linter](https://github.com/super-linter/super-linter/blob/main/README.md) via https://github.com/mundialis/github-workflows/pull/27

- added config files for markdownlint and shellcheck
- added inline exceptions for markdownlint, shellcheck and hadolint (docker)
- lint (with help of super-linter)
  - yaml (`VALIDATE_YAML`)
  - markdown (`VALIDATE_MARKDOWN`)
  - github workflows (`VALIDATE_GITHUB_ACTIONS`)
  - json (`VALIDATE_JSON`)
  - shell / bash (`VALIDATE_BASH`, `VALIDATE_BASH_EXEC`)
  - renovate (`VALIDATE_RENOVATE`)
  - Dockerfiles (`VALIDATE_DOCKERFILE_HADOLINT`)
- Linter were added but no adjustments needed for `VALIDATE_XML`, `VALIDATE_CSS`, `VALIDATE_HTML`, `VALIDATE_JAVASCRIPT_ES`, `VALIDATE_JAVASCRIPT_STANDARD`, `VALIDATE_POWERSHELL`
- ace READMEs which were deleted were (already) moved to actinia-org/ace
- merged scripts on how to create swagger docs

### CODE REVIEW

File: docker/actinia-core-dev/Dockerfile [hunk 1]
```diff
@@ -9,8 +9,8 @@ ENV FLASK_ENV=development
 ENV FLASK_DEBUG=1
 ENV DEFAULT_CONFIG_PATH=/etc/default/actinia
 
-RUN pip3 uninstall actinia-core -y
-RUN pip3 uninstall actinia_core -y
+RUN pip3 uninstall actinia-core -y \
+    RUN pip3 uninstall actinia_core -y
```
Comment (anikaweinmann) [inline docker/actinia-core-dev/Dockerfile:13]: Why is it like this and not like in the test-Dockerfile like ` &&  \`?

File: docker/actinia-core-tests/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM mundialis/actinia-core:latest as actinia_test
+FROM mundialis/actinia-core:4.14.1 as actinia_test
```
Comment (anikaweinmann) [inline docker/actinia-core-tests/Dockerfile:1]: do you have to adjust this with every release?

File: logging.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
-Actinia supports logging to fluentd as well as to stdout.
+<!-- markdownlint-configure-file { "MD013": { "line_length": 1150 } } -->
+# Actinia supports logging to fluentd as well as to stdout
 
-## tl;dr:
+## tl;dr
```
Comment (anikaweinmann) [inline logging.md:4]: What does `tl:dr` mean?

**Review by anikaweinmann — APPROVED** [review]
Thank you. Just a few little things Ii noticed. Please have a look at the failing tests.

File: docker/actinia-core-dev/Dockerfile (hunk 1 above)
Comment (mmacata) [reply docker/actinia-core-dev/Dockerfile]: good catch, updated it.

File: docker/actinia-core-tests/Dockerfile (hunk 2 above)
Comment (mmacata) [reply docker/actinia-core-tests/Dockerfile:1]: Yes and no - renovate will take care of it.

**Pull request merged**